src/Cronofy/Requests/AddToCalendarRequest.cs

```diff
@@ -0,0 +1,80 @@
+﻿namespace Cronofy.Requests
+{
+    using System;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Class for the serialization of an add to calendar request.
+    /// </summary>
+    public sealed class AddToCalendarRequest
+    {
+        /// <summary>
+        /// Gets or sets the client id for the request.
+        /// </summary>
+        /// <value>
+        /// The client id for the request.
+        /// </value>
+        [JsonProperty("client_id")]
+        public string ClientId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the client secret for the request.
+        /// </summary>
+        /// <value>
+        /// The client secret for the request.
+        /// </value>
+        [JsonProperty("client_secret")]
+        public string ClientSecret { get; set; }
+
+        /// <summary>
+        /// Gets or sets the oauth details for the request.
+        /// </summary>
+        /// <value>
+        /// The oauth details for the request.
+        /// </value>
+        [JsonProperty("oauth")]
+        public OAuthDetails OAuth { get; set; }
+
+        /// <summary>
+        /// Gets or sets the event details for the request.
+        /// </summary>
+        /// <value>
+        /// The event details for the request.
+        /// </value>
+        [JsonProperty("event")]
+        public UpsertEventRequest UpsertEventRequest { get; set; }
```

Comment:
I would name the property `Event` so it mirrors the JSON property more directly.